cmd/build.go

```diff
@@ -24,7 +25,12 @@ func setupBuildCommand() *cobra.Command {
 func buildCommandAction(cmd *cobra.Command, args []string) error {
 	cmd.Println("Build the package")
 
-	err := builder.BuildPackage()
+	err := docs.UpdateReadme()
+	if err != nil {
+		return errors.Wrapf(err, "updating %s file failed", docs.ReadmeFile)
+	}
+
+	err = builder.BuildPackage()
 	if err != nil {
 		return errors.Wrap(err, "building package failed")
 	}
```

Comment:
This is a more general comment, applicable to all `elastic-package` commands I suppose. 

When a command completes successfully, how do you feel about printing to STDOUT any side effects the command has had so the user knows where to look for things changed/created by the command? For example, lets take this command here: `elastic-package build`. Now it has two side effects: update the README and build the package. WDYT about printing for the user that the command completed and did these two things, maybe even showing the relevant paths?

Reply:
Ok, it sounds as a reasonable idea. I added relevant `Println`s.

For reference:

```
$ elastic-package build
elastic-package has been installed.
Build the package
README.md file rendered: /Users/marcin.tojek/go/src/github.com/elastic/elastic-package/test/packages/apache/docs/README.md
Package built: /Users/user/go/src/github.com/elastic/elastic-package/build/integrations/apache/0.1.4
Done
$ cd ../nginx
$ cd elastic-package build
Build the package
Package built: /Users/user/go/src/github.com/elastic/elastic-package/build/integrations/nginx/0.2.4
Done
```